src/stan/lang/grammars/semantic_actions_def.cpp

```diff
@@ -492,11 +485,12 @@ namespace stan {
              function_signature_t> >::const_iterator it
              = existing.begin();
            it != existing.end();
-           ++it)
+           ++it) {
         if (name_sig.first == (*it).first
```

Comment:
disagree here - `return true` short-circuits the loop that the condition is in.  otherwise, keep going in loop - return false only if loop terminates normally.

Reply:
You're right---missed the loop here.